frontend/benefit/handler/src/components/applicationList/useApplicationList.ts

```diff
@@ -27,50 +25,47 @@ const useApplicationList = (
   status: APPLICATION_STATUSES[]
 ): ApplicationListProps => {
   const { t } = useTranslation();
-  const { data, error, isLoading } = useApplicationsQuery(status);
+  const query = useApplicationsQuery(status);
 
-  useEffect(() => {
-    if (error) {
-      hdsToast({
-        autoDismissTime: 5000,
-        type: 'error',
-        labelText: t(`${translationsBase}.errors.fetch.label`),
-        text: t(`${translationsBase}.errors.fetch.text`, {
-          status: status,
-        }),
-      });
-    }
-  }, [t, error, status]);
+  const list = query.data?.reduce<ApplicationListItemData[]>(
```

Comment:
map didnt work?

Reply:
we have this implementation in applicant application list, but i can refactor

Reply:
fixed